src/test/scala/com/microsoft/hyperspace/index/IndexManagerTests.scala

```diff
@@ -251,6 +251,67 @@ class IndexManagerTests extends HyperspaceSuite with SQLHelper {
     }
   }
 
+  test("Verify refresh-incremental rebuild of index.") {
```

Comment:
nit:

```suggestion
  test("Verify refresh-incremental (append-only) should index only newly appended data.") {
```

Does this make sense? I wrote append-only to accommodate the merge phase between you and @pirz.